content/docs/use-cases/model-registry.md

```diff
@@ -2,68 +2,63 @@
 
 A **model registry** is a tool to catalog ML models and their versions. Models
 from your data science projects can be discovered, tested, shared, deployed, and
-audited from there. [DVC](/doc), [GTO], and [MLEM] enable these capabilities on
-top of Git, so you can stick to an existing software engineering stack. No more
+audited from there. [DVC], [GTO], and [MLEM] enable these capabilities on top of
+Git, so you can stick to an existing software engineering stack. No more
```

Comment:
Building on top of Git sounds good, but let's drop the mentions of our tools (DVC, GTO, MLEM) here and focus on it from Studio first.